definitions/infra-harbor/definition.yml

```
@@ -0,0 +1,18 @@
+domain: INFRA
+type: HARBORREGISTRY
+
+synthesis:
+  rules:
+  - identifier: hostname
+    encodeIdentifierInGUID: true
+    name: hostname
+    conditions:
+    - attribute: metricName
+      prefix: harbor_
+
+configuration:
+  alertable: true
+
+dashboardTemplates:
```

Could we try the one below, I think it will match the validation ([docs here](https://github.com/newrelic/entity-definitions/blob/4daccc4514f1a05f0c1d621ae067169ba71eb30e/docs/entity_summary.md#multiple-sources-of-data)):
```suggestion
dashboardTemplates:
  prometheus:
    template: prometheus_dashboard.json
```

I am seeing a validation error here:

```
../definitions/infra-harbor/dashboard.json invalid
[
  {
    "instancePath": "",
    "schemaPath": "#/additionalProperties",
    "keyword": "additionalProperties",
    "params": {
      "additionalProperty": "variables"
    },
    "message": "must NOT have additional properties"
  }
]
```

---

Change made and file renamed to match